test/integration/serverless/log_normalize.py

```diff
@@ -42,6 +42,9 @@ def normalize_logs(stage, aws_account_id):
         # TODO: these messages may be an indication of a real problem and
         # should be investigated
         "TIMESTAMP http: proxy error: context canceled",
+        # these are related to datadog-agent/pull/34351
```

Comment:
Here is the exact line where the logs are coming from, didn't bother figuring out the root cause but here it is in case anyone needs it in the future

https://github.com/DataDog/datadog-agent/blame/7877501df1d24ee2400938dead31bac9d5035309/comp/logs/agent/config/endpoints.go#L273